op.c

```diff
@@ -768,11 +768,10 @@ Perl_allocmy(pTHX_ const char *const name, const STRLEN len, const U32 flags)
     PERL_ARGS_ASSERT_ALLOCMY;
 
     if (flags & ~SVf_UTF8)
-        croak("panic: allocmy illegal flag bits 0x%" UVxf,
-                   (UV)flags);
```

Comment:
@mauke, am I correct in thinking that the only redundant casts you're fixing are the `(UV)` in the line above and (U8*) in line 775?  If so, then once someone (not me) okays that, this p.r. is good to go.

Reply:
Yes.